### PR TITLE
Added file icons

### PR DESCRIPTION
**Pull request by kdserra**
demo

### CODE REVIEW

**Comment by pdesaulniers** [comment]
Thanks for the PR! :)

I wish we could extend the user's icon theme with the custom icons, instead of exposing a separate "Teal Icons" theme. This PR seems to implement this idea: https://github.com/microsoft/vscode/pull/118846

File: package.json [hunk 1]
```diff
@@ -1,80 +1,93 @@
...
+		"@types/shell-quote": "^1.7.0",
+		"@types/tmp": "^0.2.0",
+		"@types/vscode": "^1.52.0",
+		"mocha": "^8.3.0",
+		"ovsx": "0.1.0-next.a9154dc",
+		"tree-sitter-cli": "^0.18.0",
+		"tree-sitter-teal": "github:euclidianAce/tree-sitter-teal#f6eb1b2",
+		"typescript": "^4.1.3",
+		"vsce": "^1.85.0"
+	},
```
Comment (pdesaulniers) [inline package.json:79]: It looks like the dependencies have been downgraded.

I suppose your changes are based on the latest .vsix version of the extension? Can you please base your changes on the current 'master' branch of the repo instead?

**Review by pdesaulniers — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (kdserra) [reply package.json:78]: Sorry about that.  I used the latest version from the extension.  I have updated it to be based off the current main branch.  Should be resolved now.

**Comment by pdesaulniers** [comment]
FYI, it is now possible to associate default icons for a language in vscode. It is not necessary to create a separate icon theme anymore :)

I have implemented this in https://github.com/teal-language/vscode-teal/commit/b9729a3e18759e72cea600a76053be5616c0b35b.

I made the logo green to make it easier to differentiate from the Lua logo. It's not technically correct, but I think it looks nice :)

image

**Pull request merged**